### DISCUSSION TITLE
Is it possible to train with the generated dataset?

### DISCUSSION THREAD

**Discussion by heroooooo** [ultralytics/yolov5#8082] [Q&A]
I have a lot of png images with alpha channels, with only one image in each class, such as:
https://i.stack.imgur.com/7baV8.png
https://i.stack.imgur.com/c06PK.png
https://i.stack.imgur.com/oKhSU.png
https://i.stack.imgur.com/3Wv4K.png
I want to detect these individual images into another image, like this one:
https://i.stack.imgur.com/GSF8x.png
The question is whether YOLO can do this task after training on a custom dataset and if it's feasible, is it correct for me to combine them into thousands of different background pictures and label them as training sets? If not, what can I do? Thanks.

**Comment by glenn-jocher** [comment]
@heroooooo for best results you want to place these on larger images, i.e. randomly place a hundred of these on a larger 640x640 images, ideally using images with different backgrounds.
